examples/stable/test/property/ic_api/json/src/index.ts

```diff
@@ -0,0 +1,54 @@
+import { IDL, jsonParse, jsonStringify, query, update } from 'azle';
```

Comment:
This is totally wrong. You should use the stable API class syntax. You can find many examples of how to do this in the examples/stable directory. You had it correct then changed it.